test/unit/migrate_assets_to_asset_manager_test.rb

```diff
@@ -95,6 +95,10 @@ class AssetFilePathsTest < ActiveSupport::TestCase
     assert @subject.respond_to?(:join)
   end
 
+  test 'delegates size to file_paths' do
```

Comment:
Minor: I might say this should be more like "implements size method", because it's not actually testing the delegation.